runtime/nls/cfre/cfrerr.nls

```diff
@@ -1559,3 +1559,10 @@ J9NLS_CFR_ERR_VALUE_TYPES_IS_NOT_SUPPORTED_V1.explanation=ValueTypes is an exper
 J9NLS_CFR_ERR_VALUE_TYPES_IS_NOT_SUPPORTED_V1.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_VALUE_TYPES_IS_NOT_SUPPORTED_V1.user_response=Contact the provider of the class file for a corrected version, or remove the -XX:+EnableValhalla option and retry.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_BC_JUMP_RECURSIVE=A jsr instruction illegally points to its own bytecode index
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_VALUE_TYPES_IS_NOT_SUPPORTED_V1.explanation=The JVM is unable to flatten the code block for a jsr instruction that refers to its own bytecode index.
+J9NLS_CFR_ERR_VALUE_TYPES_IS_NOT_SUPPORTED_V1.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_VALUE_TYPES_IS_NOT_SUPPORTED_V1.user_response=Contact the provider of the class file for a corrected version.
```

Comment:
The keys for these samples should be `J9NLS_CFR_ERR_BC_JUMP_RECURSIVE`, not `J9NLS_CFR_ERR_VALUE_TYPES_IS_NOT_SUPPORTED_V1`.
@tajila Will you fix that or would you like me to do it?

Reply:
Note this has been merged into 0.28 as well.

Reply:
I haven't heard anything from @tajila, so I created #13440 and #13441.